src/RedirectionForm.php

```diff
@@ -180,11 +179,15 @@ public function render() : string
      *
      * @param $options
      *
-     * @return \Illuminate\Http\JsonResponse
+     * @return string|\Illuminate\Http\JsonResponse
      */
-    public function toJson($options = JSON_UNESCAPED_UNICODE) : \Illuminate\Http\JsonResponse
+    public function toJson($options = JSON_UNESCAPED_UNICODE)
     {
-        return Response::json($this)->setEncodingOptions($options);
+        if (class_exists('\Illuminate\Support\Facades\Response')) {
+            return \Illuminate\Support\Facades\Response::json($this)->setEncodingOptions($options);
+        }
+
```

Comment:
I think `Illuminate` must be removed completely. 

we MUST send an `application/json` header using pure PHP.

Laravel will **overwrite output headers** if it is needed.
as you see this class implements `JsonSerializable`.
frameworks could check if the response is, type of `JsonSerializable` or not! and send related headers.

Reply:
You are right. Returning the JsonSerializable causes Laravel to add `application/json`. So calling `toJson` is not required.

Reply:
So the previous implementation of this method was correct but it should be mentioned in the documentation of `shetabit/payment`. Following code works very well in Laravel.
```
return Payment::purchase(
    (new Invoice)->amount(1000), 
    function($driver, $transactionId) {

	}
)->pay();
```
I think adding `application/json` header should be considered by the developer itself rather than adding it in the method but documentation could get improved.